.github/workflows/main.yml

```diff
@@ -7,6 +7,10 @@ on:
   repository_dispatch:
 
 jobs:
+  trigger-site-repo-workflow:
+    runs-on: ubuntu-latest
+    steps:
+    - run: gh workflow run main.yml --ref master --repo haxeflixel/haxeflixel.com
```

Comment:
is the idea that we would keep this change solely on master and not on any other branch? is there instead a way to set this so that it is only run on master, even if this change is applied to dev as well?

I try not to have master ahead of dev, and I routinely back-merge master into dev for various for various reasons, namely because people keep accidently making PRs to master and I don't notice it in time

Reply:
it can run on both if we want, really the way that the haxeflixel.com repo works, is it gets this repo on the fly when it builds the site, and doesnt merge anything from it. So if we call the github action here, it would be in the same state as it was before, except with "refreshed" flixel-docs. no commits made to the repo or anything needed, it just basically asks the site to rebuild itself

Reply:
basically all we need to do to "refresh" the documentation on haxeflixel.com, whether its dev branch OR main, is trigger the haxeflixel/haxeflixel.com github action, whether its a push, or in this case, just telling it to trigger. running the action doesnt explicitly make a commit anywhere 💪